Sources/URLEncodedForm/Codable/URLEncodedFormEncoder.swift

```diff
@@ -11,6 +11,22 @@
 /// See [Mozilla's](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/POST) docs for more information about
 /// url-encoded forms.
 public final class URLEncodedFormEncoder: DataEncoder {
+    /// The formatting of the output data.
+    public struct OutputFormatting : OptionSet {
```

Comment:
JSON coder calls it "writing options". We should probably follow suit.

Reply:
Maybe we're looking at different things? This code is copied from https://github.com/apple/swift/blob/master/stdlib/public/SDK/Foundation/JSONEncoder.swift where it is called:

```
/// The output format to produce. Defaults to `[]`.
open var outputFormatting: OutputFormatting = []
```

Reply:
Small change here:
```diff
- OutputFormatting : OptionSet
+ OutputFormatting: OptionSet
```